docs/nodes/metrics.md

```diff
@@ -18,40 +18,53 @@ Listen address can be changed in the config file (see
+| consensus_fast_syncing                  | gauge     |                 | either 0 (not fast syncing) or 1 (syncing)                                                                                                 |
+| consensus_state_syncing                 | gauge     |                 | either 0 (not state syncing) or 1 (syncing)                                                                                                |
+| consensus_block_size_bytes              | Gauge     |                 | Block size in bytes                                                                                                                        |
+| consensus_step_duration                 | Histogram | step            | Histogram of durations for each step in the consensus protocol                                                                             |
+| consensus_block_gossip_receive_latency  | Histogram |                 | Histogram of time taken to receive a block in seconds, measure between when a new block is first discovered to when the block is completed |
+| consensus_block_gossip_parts_received   | Counter   | matches_current | Number of block parts received by the node                                                                                                 |
+| consensus_quorum_prevote_delay          | Gauge     |                 | Interval in seconds between the proposal timestamp and the timestamp of the earliest prevote that achieved a quorum                        |
+| consensus_full_prevote_delay            | Gauge     |                 | Interval in seconds between the proposal timestamp and the timestamp of the latest prevote in a round where all validators voted           |
+| consensus_proposal_timestamp_difference | Histogram |                 | Difference between the timestamp in the proposal message and the local time of the validator at the time it received the message           |
+| consensus_vote_extension_receive_count  | Counter   | status          | Number of vote extensions received                                                                                                         |
+| consensus_proposal_receive_count        | Counter   | status          | Total number of proposals received by the node since process start                                                                         |
+| consensus_proposal_create_count         | Counter   |                 | Total number of proposals created by the node since process start                                                                          |
+| consensus_round_voting_power_percent    | Gauge     | vote_type       | A value between 0 and 1.0 representing the percentage of the total voting power per vote type received within a round                      |
```

Comment:
I think I would be super-useful for node operators to have info on:
* Number of proposals rejected by the App (`ProcessProposal` returning `reject`)
* Number of vote extensions rejected by the App (`VerifyVoteExtension` returning `reject`)

Reply:
These are signs that, either byzantine validators, or the App's logic being flawed (non-deterministic, but not only)

Reply:
Oh, I see below that both metrics that an "accepted" and "rejected" status, so I guess you can ignore my comment above... sorry for the noise